src/Pages/Query.php

```diff
@@ -108,6 +108,18 @@ public function get($throwOnMissing = false)
                 return $this->repository->load($type, $name, $this->locale, $throwOnMissing);
             });
     }
+    
+    
+    /**
+     * Mimic eloquent's Builder and execute the query
+     *
+     * @param bool $throwOnMissing
+     * @return Illuminate\Support\Collection
+     */
+    public function exists($throwOnMissing = false)
+    {
+        return true;
```

Comment:
This returns 'true' all the time despite this instance exists or not. It should query the database and return a proper boolean value according to the existence of the model instance.

Reply:
For this check, could we utilize the already mimic'd `get()`?
```suggestion
        $results = $this->get($throwOnMissing);

        return $results->count() === 1;
```

Reply:
@AbdullahGhanem can you check this comment and apply it?